punch/action_register.py

```diff
@@ -0,0 +1,22 @@
+from punch.actions import increase_part
+from punch.actions import set_part
+from punch.actions import conditional_reset
+
+
+class ActionRegister:
```

Comment:
Your setup.py claims that `punch` supports python2; if so, then this syntax is wrong; on python2 it would make an "old-style class" which does not inherit from `object`, so you need to explicitly inherit; `class ActionRegister(object):`

The same for all other classes declared in here.

You should also at least add `from __future__ import print_function, absolute_import, division` to all files so that the behavior of both versions is closer to the same.

Reply:
:+1: